kubernetes/linux/setup.sh

```diff
@@ -85,3 +85,7 @@ rm -f $TMPDIR/docker-cimprov*.sh
 rm -f $TMPDIR/azure-mdsd*.deb
 rm -f $TMPDIR/mdsd.xml
 rm -f $TMPDIR/envmdsd
+
+# Remove settings for cron.daily that conflict with the node's cron.daily. Since both are trying to rotate the same files
+# in /var/log at the same time, the rotation doesn't happen correctly and then the *.1 file is forever logged to.
+rm /etc/logrotate.d/alternatives /etc/logrotate.d/apt /etc/logrotate.d/azure-mdsd /etc/logrotate.d/rsyslog
```

Comment:
Grace, please create a workitem in the april release, to make /var/log readonly for a future fix.